lib/types.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 
 const Gql = require('graphql-tag');
+const cloneDeep = require('lodash.clonedeep');
```

Comment:
added lodash.clonedeep as a dependency, hope that's ok

Reply:
nit: `CloneDeep` for consistent code style.